src/content/deployment/flavors.md

```diff
@@ -52,32 +58,63 @@ Open your project in Xcode.
 </li>
 <li>
 
-Select **Product** > **Scheme** > **New Scheme** from the menu to
-add a new `Scheme`.
+Navigate to the **Build Settings** of the target(s)
+and follow these instructions:
+
+</li>
+
+If `DART_DEFINES` doesn't exist yet:
+
+* Click the **+** button in the top left.
+* Select **Add User-Defined**.
+* Enter `DART_DEFINES` as the name.
+* Enter the **base64 encoded version**
```

Comment:
Wikipedia seems to always capitalize "Base64", should we as well?

See: https://en.wikipedia.org/wiki/Base64

```suggestion
* Enter the **Base64 encoded version**
```